templates/typesetting/typesetting_review_assignment.html

```diff
@@ -84,7 +85,7 @@ <h2>Manage Assignment</h2>
                         {{ form.assignment|foundation }}
                     </div>
                   <button class="button success" name="edit">Save Changes</button>
-                  <button class="button alert" name="delete">Delete Assignment</button>
+                  <button class="button alert" name="delete" onclick="return confirm('Are you sure you want to delete this assignment?');">Delete Assignment</button>
```

Comment:
This is a matter of personal preference, but worth using a modal instead of a prompt? The browser prompts break the UI experience and they can be terribly annoying (as well as disabled by the user)